src/vmm/src/lib.rs

```diff
@@ -202,6 +202,8 @@ pub struct Vmm {
     mmio_device_manager: MMIODeviceManager,
     #[cfg(target_arch = "x86_64")]
     pio_device_manager: PortIODeviceManager,
+
+    _smbios_oem_strings: Option<Vec<String>>,
```

Comment:
Let's avoid adding a new field to `Vmm`. Instead, please extend `configure_system` to accept a third argument with `smbios_oem_strings`.